sketch.test.js

```diff
@@ -1,5 +1,21 @@
-const sum = require('./sketch');
+const math = require('./sketch');
 
 test('adds 1 + 2 to equal 3', () => {
-    expect(sum(1, 2)).toBe(3);
+    expect(math.sum(1, 2)).toBe(3);
+}); 
+
+test('adds 4 - 1 to equal 3', () => {
+    expect(math.rest(2, 1)).toBe(3);
+}); 
+
```

Comment:
it seems the problems are with the test case it should be `math.rest(4, 1)` instead of `math.rest(2, 1)`